README.md

```diff
@@ -1,38 +1,98 @@
-# VIPR. Verifying Integer Programming Results
+# VIPR: Verifying Integer Programming Results
 
 ## About
 
 *VIPR* is new a software project to verify, in exact rational arithmetic, the correctness of results computed by mixed-integer linear programming solvers.  It is based on an elementary file format for LP-based branch-and-cut certificates proposed in the article
 
 > Kevin K.H. Cheung, Ambros Gleixner, and Daniel E. Steffy: [Verifying Integer Programming Results](http://dx.doi.org/10.1007/978-3-319-59250-3_13). In: F. Eisenbrand and J. Koenemann, eds., Integer Programming and Combinatorial Optimization: 19th International Conference, IPCO 2017, pp. 148-160, 2017, [`doi:10.1007/978-3-319-59250-3_13`](http://dx.doi.org/10.1007/978-3-319-59250-3_13).
 
-This repository contains a detailed technical [specification of the certificate file format](http://rawgit.com/ambros-gleixner/VIPR/master/cert_spec_v1_0.html), [software](code/) to check, display, and compress certificate files, and [supplementary information](experiments/) on the computational experiments conducted for the article above.
+This repository contains a detailed technical specification of the certificate file format in [Version 1.0](cert_spec_v1_0.html) and [Version 1.1](cert_spec_v1_1.html), [software](code/) to check, display, compress, and complete certificate files, and [supplementary information](experiments/) on the computational experiments conducted for the article above.
```

Comment:
describe the extension of lin by incomplete and weak in v1.1

Reply:
I would also completely remove the .html files and change the links to markdown (which now supports mathjax on github as well, so it is more readable than the html I think

Reply:
Happily!